luggage_aliases.module

```diff
@@ -40,8 +40,28 @@ function _luggage_aliases_people_node_form_submit($form, &$form_state) {
   }
 }
 
+/**
+ * Generate alias name.
+ */
 function _luggage_aliases_getTermName($form_state) {
-  $title = $form_state['values']['field_people_title'][LANGUAGE_NONE][0]['value'] . ' ' .  $form_state['values']['field_people_first_name'][LANGUAGE_NONE][0]['value'] . ' ' . $form_state['values']['field_people_middle_initial'][LANGUAGE_NONE][0]['value'] . ' ' . $form_state['values']['field_people_last_name'][LANGUAGE_NONE][0]['value'];
+  $title = '';
```

Comment:
Since we have title (as part of a person's name) and title (as the title of the term) It might be good to change this to $term_title or $term_name? Kinda meh about fixing this tho